campaignion_tracking/js/campaignion_tracking.js

```diff
@@ -8,7 +8,7 @@ Drupal.behaviors.campaignion_tracking.attach = function(context, settings) {
    * Nothing happens if the tracker is not available.
    */
 
-  function gracefulDispatch (name, data, trackingContext) {
+  function gracefulDispatch (type, name, data, trackingContext) {
```

Comment:
This could just as well be defined outside the behavior which makes the behavior simpler and makes it clear it’s pure function (not a closure).

Reply:
I think it makes it harder find the Drupal behavior… :thinking:

Reply:
> I think it makes it harder to find the Drupal behavior…

It could also be below the behavior. Another method of the behaviour would also be an improvement.

Reply:
Much better. Thanks.